db/dataset_eval.py

```diff
@@ -68,8 +68,9 @@ def evaluate_dataset(dataset_id, normalize, eval_location, filter_type=None):
             raise JobExistsException
 
         # Validate dataset contents
-        validate_dataset_contents(db.dataset.get(dataset_id))
-        return _create_job(connection, dataset_id, normalize, eval_location, filter_type)
+        ignore_rec = validate_dataset_contents(db.dataset.get(dataset_id))
```

Comment:
We have two types of dataset validation. One to check that each class has enough recordings (`validate_dataset_structure`), and another to check if the recordings are in AB. If some recordings are ignored, then the number if items in the class may have fallen below our threshold. We should check again after removing these recordings

Reply:
I add additional check to `evaluate_dataset` function